examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
<blockquote>to implement <strong>Sentinel2-A (S2A)</strong> and <strong>Sentinel2-B (S2B)</strong>.</blockquote><p>-&gt; to simulate Sentinel2-A (S2A) and Sentinel2-B (S2B) performing a task.</p><p> </p><p>For rasterio , ideally say how to install?</p><p> </p><p>pip install rasterio? or conda?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<blockquote>to implement <strong>Sentinel2-A (S2A)</strong> and <strong>Sentinel2-B (S2B)</strong>.</blockquote><p>-&gt; to simulate Sentinel2-A (S2A) and Sentinel2-B (S2B) performing a task.</p>